integration/spark/shared/build.gradle

```diff
@@ -62,6 +62,7 @@ dependencies {
     testFixturesApi(platform("org.junit:junit-bom:${junit5Version}"))
     testFixturesApi("org.junit.jupiter:junit-jupiter")
     testFixturesApi("org.junit.jupiter:junit-jupiter-params")
+    testFixturesApi("org.junit-pioneer:junit-pioneer:1.9.1")
```

Comment:
is there anything in tests that uses it?

Reply:
```
  @SetEnvironmentVariable(key = "AWS_DEFAULT_REGION", value = "us-west-2")
```